modules/avm/variables.tf

```
@@ -13,6 +37,12 @@ variable "aws_config" {
   description = "AWS Config settings"
 }
 
+variable "create_account_password_policy" {
+  type        = bool
+  description = "Define if the AWS account password policy should be created"
+  default     = true
```

```suggestion
  default     = true
  description = "Set to false to prevent the AWS account policy from being created"
```

I swapped the `default` and `description` params, and usually we say `Set to X to ...`, so amended the description.

---

agree